Sources/LiveViewNative/Modifiers/Text Input and Output Modifiers/MultilineTextAlignmentModifier.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  MultilineTextAlignmentModifier.swift
+//  LiveViewNative
+//
+//  Created by Dylan.Ginsburg on 4/18/23.
+//
+
+import SwiftUI
+
+/// Sets the alignment of a text view that contains multiple lines of text.
+///
+/// ```html
+/// <Text
+///     modifiers={
+///         multiline_text_alignment(@native, alignment: :trailing)
+///     }
+/// >
+///   Hello world.
+///   Goodbye cruel world.
+/// </Text>
+/// ```
+///
+/// ## Arguments
+/// * ``alignment``
+#if swift(>=5.8)
+@_documentation(visibility: public)
+#endif
+struct MultilineTextAlignmenteModifier: ViewModifier, Decodable, Equatable {
+    /// One of the `TextAlignment` enumerations.
```

Comment:
I think there should be a list of possible values:
```suggestion
    /// One of the `TextAlignment` enumerations.
    ///
    /// Possible values:
    /// * `center`
    /// * `leading`
    /// * `trailing`
```